packages/cli/src/eventbus/MessageEventBus/MessageEventBus.ts

```diff
@@ -225,7 +223,7 @@ export class MessageEventBus extends EventEmitter {
 
 	private async emitMessage(msg: EventMessageTypes) {
 		if (config.getEnv('endpoints.metrics.enable')) {
-			await incrementPrometheusMetric(msg);
+			this.emit(MetricsCounterEvents.messageEventBusEvent, msg);
```

Comment:
shouldn't the names of the events emitted by the event-bus be defined along side the event-bus code? en event-emitter picking names of the events from another package/folder feels like an anti-pattern.